src/EditorFeatures/CSharpTest2/Recommendations/KeywordRecommenderTests.cs

```diff
@@ -2,22 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
+using Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders;
+using System.Reflection;
 using System;
 using System.Collections.Generic;
-using System.Reflection;
-using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders;
+using System.Threading.Tasks;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
 {
     public class KeywordRecommenderTests : RecommenderTests
     {
-        private static readonly Dictionary<SyntaxKind, AbstractSyntacticSingleKeywordRecommender> s_recommenderMap =
-            new Dictionary<SyntaxKind, AbstractSyntacticSingleKeywordRecommender>(SyntaxFacts.EqualityComparer);
+        private static readonly Dictionary<SyntaxKind, AbstractSyntacticSingleKeywordRecommender> s_recommenderMap = new(SyntaxFacts.EqualityComparer);
+        private readonly string computedKeywordText;
+
+        protected override string KeywordText => computedKeywordText;
```

Comment:
```suggestion
        protected override string KeywordText { get; }
```

?

(not really getting why we need the field here).

Reply:
Was gonna say "Because adding `set` will lead to a compiler error", but then realized, that I don't need `set` here because initializing is taking place in the constructor 🤦‍♂️